Rakefile

```diff
@@ -2,7 +2,7 @@ $LOAD_PATH << File.expand_path('../lib', __FILE__)
 require 'fileutils'
 require 'bundler/gem_tasks'
 require 'rspec/core/rake_task'
-
+require 'tempfile'
```

Comment:
What is this for again?

Reply:
The testbed rake task uses Tempfile but for some reason the gem wasn't being required, so it would raise an error when trying to run the specs.